openshift/release/update-to-head.sh

```diff
@@ -48,7 +48,7 @@ if hash hub 2>/dev/null; then
    COUNT=$(hub api -H "Accept: application/vnd.github.v3+json" repos/${REPO_OWNER_NAME}/${REPO_NAME}/pulls --flat \
     | grep -c "${message}") || true
    if [ "$COUNT" = "0" ]; then
-      hub pull-request -m "${message}" -l "kind/sync-fork-to-upstream" -b ${REPO_OWNER_NAME}/${REPO_NAME}:${REPO_BRANCH} -h ${REPO_OWNER_NAME}/${REPO_NAME}:${REPO_BRANCH_CI}
+      hub pull-request -m "${message}" -l "kind/sync-fork-to-upstream,approved,lgtm" -b ${REPO_OWNER_NAME}/${REPO_NAME}:${REPO_BRANCH} -h ${REPO_OWNER_NAME}/${REPO_NAME}:${REPO_BRANCH_CI}
```

Comment:
```suggestion
      hub pull-request -m "${message}" -l "kind/sync-fork-to-upstream,approved,lgtm" -b "${REPO_OWNER_NAME}"/"${REPO_NAME}":"${REPO_BRANCH}" -h "${REPO_OWNER_NAME}"/"${REPO_NAME}":"${REPO_BRANCH_CI}"
```

Not this PR's fault but it would be great if you could fix the lint error.

Reply:
Never mind. There are so many same issue on this script so I think it is okay to defer to another PR for the lint fix.